pkg/ebpf/events_definitions.go

```diff
@@ -5850,7 +5850,11 @@ var EventsDefinitions = map[int32]EventDefinition{
 	MagicWriteEventID: {
 		ID32Bit: sys32undefined,
 		Name:    "magic_write",
-		Probes:  []probe{}, Sets: []string{},
+		Probes: []probe{
```

Comment:
you've added `magic_write` event to be able to capture by itself. therefore, you should set it as essential if capture write was chosen by the user (https://github.com/aquasecurity/tracee/blob/main/pkg/ebpf/tracee.go#L203).
also keep track of the capture here https://github.com/aquasecurity/tracee/blob/main/pkg/ebpf/events_processor.go#L107

Reply:
nice catches @roikol